orderer/common/server/main.go

```diff
@@ -806,42 +805,23 @@ func initializeMultichannelRegistrar(
 
 	consenters := map[string]consensus.Consenter{}
 
-	var icr etcdraft.InactiveChainRegistry
 	if conf.General.BootstrapMethod == "file" || conf.General.BootstrapMethod == "none" {
 		if bootstrapBlock != nil && isClusterType(bootstrapBlock, bccsp) {
 			// with a system channel
-			etcdConsenter := initializeEtcdraftConsenter(consenters, conf, lf, clusterDialer, bootstrapBlock, repInitiator, srvConf, srv, registrar, metricsProvider, bccsp)
-			icr = etcdConsenter.InactiveChainRegistry
```

Comment:
I don't understand why this:
```
			etcdConsenter := initializeEtcdraftConsenter(consenters, conf, lf, clusterDialer, bootstrapBlock, repInitiator, srvConf, srv, registrar, metricsProvider, bccsp)
```

was changed. 

Seems like now `icr = etcdConsenter.InactiveChainRegistry` was removed. Don't we need it?

Reply:
No, because this reference was only used in kafka. The "real" ICR is internal to etcd/raft now. It is also destined for deprecation because it is related to the system channel, but I'll remove it from etcdraft later.